service/experiment-config.yaml

```diff
@@ -3,7 +3,7 @@
 # will not work with your setup.
 
 trials: 20
-max_total_time: 82800  # 23 hours, the default time for preemptible experiments.
+max_total_time: 43200  # 23 hours, the default time for preemptible experiments.
```

Comment:
Please don't change this. Let's just do the full time for the experiment. It's not a big deal.

Reply:
ok,fine!